tests/testutils/_primer/test_primer.py

```diff
@@ -20,15 +20,21 @@
 
 PRIMER_CURRENT_INTERPRETER = (3, 10)
 
-
-@pytest.mark.skipif(
-    sys.platform in {"win32", "darwin"},
-    reason="Primers are internal will never be run on costly github action (mac or windows)",
+OS_SKIP_REASON = (
+    "Primers are internal will never be run on costly github action (mac or windows)"
 )
-@pytest.mark.skipif(
-    sys.version_info[:2] != PRIMER_CURRENT_INTERPRETER or IS_PYPY,
-    reason=f"Primers are internal will always be run for only one interpreter (currently {PRIMER_CURRENT_INTERPRETER})",
+INTERPRETER_SKIP_REASON = (
+    "Primers are internal will always be run for only one interpreter (currently"
+    f" {PRIMER_CURRENT_INTERPRETER})"
 )
+
+NOT_PRIMER_OS = sys.platform in {"win32", "darwin"}
+NOT_PRIMER_INTERPRETER = sys.version_info[:2] != PRIMER_CURRENT_INTERPRETER or IS_PYPY
+DEFAULT_ARGS = ["python tests/primer/__main__.py", "compare", "--commit=v2.14.2"]
+
+
+@pytest.mark.skipif(NOT_PRIMER_OS, reason=OS_SKIP_REASON)
```

Comment:
If we create a test class we can do this skip mark only once.

Reply:
[ff40caa](https://github.com/PyCQA/pylint/pull/6973/commits/ff40caa15c5716cac56923b246b24a9d0f8a28e5)